.travis.yml

```diff
@@ -17,7 +22,8 @@ virtualenv:
   system_site_packages: true
 
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
+#  - git clone https://github.com/OCA/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/savoirfairelinux/maintainer-quality-tools.git -b containers ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
```

Comment:
OCA/web shouldn't be defined there, it could be here https://github.com/OCA/vertical-ngo/blob/8.0/oca_dependencies.txt

I think it's already downloaded as a sub dependency